sound/soc/sof/ipc4-topology.c

```diff
@@ -264,7 +264,7 @@ static int sof_ipc4_get_audio_fmt(struct snd_soc_component *scomp,
 		goto err_out;
 	}
 
-	available_fmt->out_audio_fmt = out_format;
+	available_fmt->output_audio_fmts = out_format;
```

Comment:
commit message refers to wrong structure, there is no out_audio_fmt in sof_ipc4_pin_format, see https://github.com/thesofproject/linux/pull/4163/commits/78aa98f13169db39faedbb988e5ec258af5a043d#diff-d8bbc05d879b6eee2041d6fc0ee06f050be097ac05b12cfec9b35d89f66d3a84R156

You must be referring to struct sof_ipc4_available_audio_format

Reply:
fixed